assume/strategies/naive_strategies.py

```diff
@@ -137,6 +137,83 @@ def calculate_bids(
         bids = [order]
 
         bids = self.remove_empty_bids(bids)
+        return bids
+
+
+class NaiveDASteelplantStrategy(BaseStrategy):
+    def calculate_bids(
+        self,
+        unit: SupportsMinMax,
+        market_config: MarketConfig,
+        product_tuples: list[Product],
+        **kwargs,
+    ) -> Orderbook:
+        bids = []
+        start = product_tuples[0][0]  # start time of the first product
+
+        if unit.opt_power_requirement is None:
+            if unit.objective == "min_variable_cost":
+                unit.determine_optimal_operation_without_flex()
+
+        # unit.run_modified_optimization()
+
+        bids = []
+        for product in product_tuples:
+            """
+            for each product, calculate the marginal cost of the unit at the start time of the product
+            and the volume of the product. Dispatch the order to the market.
+            """
+            start = product[0]
+            volume = unit.opt_power_requirement.loc[start]
+            price = 3000
+            bids.append(
+                {
+                    "start_time": product[0],
+                    "end_time": product[1],
+                    "only_hours": product[2],
+                    "price": price,
+                    "volume": -volume,
+                }
+            )
+
+        return bids
+
+
+class NaiveRedispatchSteelplantStrategy(BaseStrategy):
+    def calculate_bids(
+        self,
+        unit: SupportsMinMax,
+        market_config: MarketConfig,
+        product_tuples: list[Product],
+        **kwargs,
+    ) -> Orderbook:
+        bids = []
+        start = product_tuples[0][0]  # start time of the first product
+
+        if unit.flex_power_requirement is None:
+            if unit.flexibility_measure == "max_load_shift":
```

Comment:
I would just call some `unit.optimal_flex_operation` or something and leave the decision of the flexibility_measure in the unit itself, not in the strategy - to determine the optimization target function

Reply:
Sorry didnt get you. We are choosing the flexibility_measure in the unit itself. In Unit we hav an attribute where we can guve a flexibility_measure to the unit.

Reply:
see a9eba5b73b64b78950e04c361d96b70d4ddfbadc which was merged into this branch :)